match/lib/match/options.rb

```diff
@@ -64,6 +64,16 @@ def self.available_options
                                      verify_block: proc do |value|
                                        ENV["FASTLANE_TEAM_ID"] = value.to_s
                                      end),
+        FastlaneCore::ConfigItem.new(key: :git_user_name,
+                                     env_name: "MATCH_GIT_USER_NAME",
+                                     description: "git user name to commit",
```

Comment:
Thank you for your review. I understand.
I will replace `user name` to `full name` and push this.